admin/installer/setup_volumehub.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Set up Volume Hub if a token has been supplied.
+set -ex
+
+: ${volumehub_token:?}
+: ${node_number:?}
+
+if test -z "${volumehub_token}"; then
+    # No Volume Hub token supplied.
```

Comment:
Is that logged somewhere? Should we get some kind of feedback or help saying how to properly call the script?